forc-plugins/forc-client/src/util/pkg.rs

```diff
@@ -85,3 +102,366 @@ pub(crate) fn built_pkgs(path: &Path, build_opts: &BuildOpts) -> Result<Vec<Arc<
 
     Ok(built_pkgs)
 }
+
+pub fn generate_chunk_loader_contract_src(
```

Comment:
I'd prefer to move all of these functions relating to contract chunking into their own module, `util/contract_chunk.rs`